code/API_definitions/CAMARA Connectivity Insights API.yaml

```diff
@@ -52,89 +75,178 @@
           description: Base path for the Connectivity Insights API
 
     tags:
-    - name: Network Monitoring Policy Management
-      description: Define KPIs for key network metrics as required for the application to ensure the end users of the application are getting the optimal user experience. KPIs such as latency, jitter, packetloss, throughput etc
     - name: Connectivity Insights
-      description: Read the network's level of confidence that it can meet an application's performance demands, for a given end user device. 
-    - name: Connectivity Insights Subscriptions
-      description: Subscribe to be notified of changes in the  network's ability to meet the application's demands, for a given device.
+      description: Read information about the network the client is currently attached to.
+    - name: Quality Insights
+      description: Read the network's level of confidence that it can meet the quality thresholds for a given application profile and end user device.
+    - name: Application Profiles
+      description: Operations to define, read and manage an application's thresholds for network quality (latency, jitter, loss, throughput)
+    - name: Notifications Management
+      description: Create and manage a subscription to receive periodic connectivity insights
    
    
     paths:
-      /network-monitoring/policy:
+      /network-info/insights:
+        get:
+          security:
+          - openId:
+            - connectivity_insights:network_info:read
+          tags:
+          - Connectivity Insights
+          description: Determine the type of network the given device is connected to, and if it is a cellular network, the network identity and connectivity type (3G/4G/5G-SA/5G-NSA)
+          operationId: readNetworkInfoInsight
+          responses:
+            "200":
+              description: OK
+              content:
+                application/json:
+                  schema:
+                      $ref: '#/components/schemas/NetworkInfoInsight'
```

Comment:
my recommendation would be to include the information within network-quality rather than having a separate API for this in network insights.

Reply:
How about `network-quality/info` as the path to get the basic information? I think it makes sense to distinguish that from the live state for a given session, because (1) it is commonly requested by developers (2) it is relatively simple to implement. It could also be included in the network-insight response as it is now.